tests/TourbusStageController.test.js

```
@@ -1,51 +1,50 @@
+        canvas: { style: {} },
+        stage: { addChild: mock.fn() },
+        screen: { width: 800, height: 600 },
+        ticker: {
+          add: currentTickerAdd,
+          remove: currentTickerRemove,
+          lastTime: 1000,
+          deltaMS: 16.6
+        },
+        init: mock.fn(() => Promise.resolve()),
+        destroy: currentAppDestroy
+      }
```

Biome (2.4.4)

[error] The constructor should not return a value.

(lint/correctness/noConstructorReturn)